package.json

```diff
@@ -18,7 +18,11 @@
   },
   "dependencies": {
     "@headlessui/react": "^1.7.15",
+    "@portal-payments/solana-wallet-names": "^2.1.0",
+    "@solana/web3.js": "^1.77.3",
+    "bigint-buffer": "^1.1.5",
     "classnames": "^2.3.2",
+    "encoding": "^0.1.13",
```

Comment:
Are `bigint-buffer` and `encoding` needed as explicit dependencies?

Reply:
Nope, removed them.